PatternPal/PatternPal.Core/Checks/ICheck.cs

```diff
@@ -115,22 +115,24 @@ internal static class CheckBuilder
     /// <param name="priority">The <see cref="Priority"/> of this <see cref="CheckCollection"/>.</param>
     /// <param name="checks">The sub-<see cref="ICheck"/>s of this <see cref="CheckCollection"/>.</param>
     /// <returns>The created <see cref="CheckCollection"/>.</returns>
-    internal static CheckCollection Any(
+    internal static NodeCheck< INode > Any(
```

Comment:
This type is unhandled in the switchcase in RunCheck in NodeCheck.cs. This causes an exception thrown by the default case. You can recreate this error by running singletonTestcase11.cs through the consoleapp. Maybe add an extra case in the switch or there is a different type for any and all needed like the removed checkcollection?

Reply:
Good point, I'll add node check to the switch.